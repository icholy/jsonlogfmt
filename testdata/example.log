time="2015-03-26T01:27:38-04:00" level=debug msg="Started observing beach" animal=walrus number=8
time="2015-03-26T01:27:38-04:00" level=info msg="A group of walrus emerges from the ocean" animal=walrus size=10
time="2015-03-26T01:27:38-04:00" level=warning msg="The group's number increased tremendously!" number=122 omg=true
time="2015-03-26T01:27:38-04:00" level=debug msg="Temperature changes" temperature=-4
time="2015-03-26T01:27:38-04:00" level=panic msg="It's over 9000!" animal=orca size=9009
