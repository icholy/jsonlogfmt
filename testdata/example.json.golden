{"animal":"walrus","level":"debug","msg":"Started observing beach","number":8,"time":"2015-03-26T01:27:38-04:00"}
{"animal":"walrus","level":"info","msg":"A group of walrus emerges from the ocean","size":10,"time":"2015-03-26T01:27:38-04:00"}
{"level":"warning","msg":"The group's number increased tremendously!","number":122,"omg":true,"time":"2015-03-26T01:27:38-04:00"}
{"level":"debug","msg":"Temperature changes","temperature":-4,"time":"2015-03-26T01:27:38-04:00"}
{"animal":"orca","level":"panic","msg":"It's over 9000!","size":9009,"time":"2015-03-26T01:27:38-04:00"}
